fontra2fontir/src/fontra.rs

```diff
@@ -13,6 +14,24 @@ use fontir::error::Error;
 use serde::Deserialize;
 use write_fonts::types::Tag;
 
+#[derive(Debug, Clone, Deserialize, Hash, PartialEq, Eq, PartialOrd, Ord)]
+pub(crate) struct AxisName(pub(crate) String);
```

Comment:
I'm really not sure in this particular case that this is an improvement over `type AxisName = String;`. 


I like new types in many cases, but generally when there is either some invariant that we want to enforce by construction (like we want to ensure that all axis names are a given length or include only characters in some allowed range) or when there is some type-specific API that we would like to provide; here it feels like the only thing we are achieving here is essentially documentation, and I think that a typedef is fine for that.


In general I also think that "newtype with a public inner type" has a bit of code smell, since it adds the additional complexity of a new type without any guarantees.